dingo/pipe/nodes/sampling_node.py

```diff
@@ -65,3 +65,7 @@ def samples_file(self):
         return os.path.join(
             self.inputs.result_directory, self.label + ".hdf5"
         )
+
+    @property
+    def result_file(self):
+        return self.samples_file
```

Comment:
Given that `self.samples_file` is used should the TODO in line 64 be removed?

Reply:
Ah, I have to check whether that's referred to. We need `result_file` for `PESummary`. But I'm not sure whether the other one is needed. I'll fix later.